kong/plugins/moesif/log.lua

```diff
@@ -430,7 +425,17 @@ function _M.execute(conf, message)
     app_configs["user_rules"] = {}
     app_configs["company_rules"] = {}
     config_hashes[hash_key] = app_configs
-    queue_hashes[hash_key] = {} 
+    queue_hashes[hash_key] = {}
```

Comment:
just some clarification on the fix here, earlier also we were calling the get_config at the start,  but after the call we are overwriting it again 
config_hashes[hash_key] = app_configs (which is set to default to 100)

we should first set to default and then call get_config function, so config_hashes[key] stays correct.